docs/kratos/guides/zero-trust-iap-proxy-identity-access-proxy.mdx

```diff
@@ -230,8 +230,8 @@ authenticators:
 
 #### Allowed Authorizer
 
-The [Allowed Authenticator](../../oathkeeper/pipeline/authz#allowed) simply allows all users to access the URL. Since we don't
-have Role-based access control (RBAC) or an Access Control list (ACL) in place for this example, this will be enough.
+The [Allowed Authenticator](../../oathkeeper/pipeline/authz#allow) simply allows all users to access the URL. Since we don't have
```

Comment:
To make docusaurus work at its best, make sure to add the file extensions too (`authz.md` or `authz.mdx`)

Reply:
some use it, some don't. whenever you see one that doesn't use it, please add it. 
I think it's not super critical but over time we can weed them out.